pom.xml

```diff
@@ -275,7 +281,7 @@
         <carbon.identity.framework.version>5.11.148</carbon.identity.framework.version>
         <carbon.identity.framework.imp.pkg.version.range>[5.0.0, 6.0.0)
         </carbon.identity.framework.imp.pkg.version.range>
-        <carbon.kernel.version>4.4.11</carbon.kernel.version>
+        <carbon.kernel.version>4.6.1</carbon.kernel.version>
```

Comment:
Need to update proper kernel version

Reply:
Fixed with [0921b83](https://github.com/wso2-extensions/identity-x509-commons/pull/23/commits/0921b83a76828a797559cbc8e7991f03fff161b5)